src/Microsoft.Dnx.Runtime.Sources/project.json

```diff
@@ -8,16 +11,7 @@
         "dnx451": { },
         "dnxcore50": {
             "dependencies": {
-                "System.Collections": "4.0.11-beta-*",
-                "System.ComponentModel": "4.0.1-beta-*",
-                "System.Console": "4.0.0-beta-*",
-                "System.Diagnostics.Debug": "4.0.11-beta-*",
-                "System.Linq": "4.0.1-beta-*",
-                "System.Reflection": "4.0.11-beta-*",
-                "System.Runtime": "4.0.21-beta-*",
-                "System.Runtime.Extensions": "4.0.11-beta-*",
-                "System.Threading": "4.0.11-beta-*",
-                "System.Threading.Tasks": "4.0.11-beta-*"
+                "System.Runtime": "4.0.21-beta-*"
```

Comment:
He's not talking about the assembly-in-sources-package issue.

VS will show you build errors unless you tell it not to build the project in the solution config.